src/test/python/test_imsc_reader.py

```diff
@@ -35,9 +35,52 @@
 import ttconv.model as model
 import ttconv.style_properties as styles
 import ttconv.imsc.reader as imsc_reader
+import ttconv.imsc.namespaces as xml_ns
 
 class IMSCReaderTest(unittest.TestCase):
 
```

Comment:
Not certain if this is needed. Happy to remove if not.

Reply:
What are you referring to?

Reply:
def setUp(self):

Reply:
Already defined in `imsc_writer`